.azure-pipelines/azure-pipelines.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
       # Run flake8 validation on a shallow clone
```

Comment:
```suggestion
      # Run Ruff validation on a shallow clone
```

Reply:
Ah, dammit, I text-searched but forgot that ripgrep ignores hidden files/folders by default, separately from `--no-ignore`.